### ISSUE TITLE
Detect available screen resolution in fullscreen mode

### ISSUE THREAD

**Issue by evertvorster** [vegastrike/Vega-Strike-Engine-Source#882]
There is a lot of complexity involved in the screen setup, and this may be a hangover from ye olden days. 

What is really needed is in the vsettings app is a screen detection routine. 
It should detect the number of screens and the resolution of those screens. 

This assumes that you want to be playing full screen and at the native resolution of the selected screen. 
This is almost always the case. 

However, if you want to be playing in windowed mode, then some common resolutions should be listed.

**Comment by royfalk** [comment]
This is definitely **not** the case with modern games. Most games can't run in 4k with a decent framerate.

 Vegastrike is an outlier here as it doesn't really use modern GPUs and CPUs properly and so can run in 4k with no problem. 

However, I can see us using more resources in a year or two and having to revert this. From a programming standpoint, something like take the native resolution probably isn't a significant effort. 

I'd like to see some discussion/polling on this before committing. Maybe on gitter?

**Comment by royfalk** [comment]
OK. See if you can find a reference in SDL as well as a discussion or code snippet to implement this. If you can find it, I'll implement this.

**Comment by BenjamenMeyer** [comment]
Yes, I'd like to see something in the VS Settings App - there's something minimal in there now; but we should be able to query and select. Same for monitors.

**Comment by evertvorster** [comment]
This is handled in the new settings app.
